dotCMS/src/test/java/com/dotcms/filters/NormalizationFilterTest.java

```diff
@@ -0,0 +1,115 @@
+package com.dotcms.filters;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+
+import com.dotcms.UnitTestBase;
+import java.io.IOException;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+
+public class NormalizationFilterTest extends UnitTestBase {
+
+    private final static NormalizationFilter normalizationFilter = new NormalizationFilter();
+    private static HttpServletResponse response;
+    private static FilterChain chain;
+    private static ArgumentCaptor<HttpServletRequest> capturedRequest;
+
+    @BeforeClass
+    public static void prepare() throws IOException, ServletException {
+
+        //Response
+        response = mock(HttpServletResponse.class);
+
+        //Chain
+        chain = mock(FilterChain.class);
+        capturedRequest = ArgumentCaptor
+                .forClass(HttpServletRequest.class);
+        //Capturing the request when is passed down to the chain
+        doNothing().when(chain).doFilter(capturedRequest.capture(), any(HttpServletResponse.class));
+    }
+
+    /**
+     * Test to verify the {@link NormalizationFilter} is applying properly the normalization on
+     * invalids URIs
+     */
+    @Test
+    public void test_uri_normalization_invalid_URI() throws IOException, ServletException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4798850)